lms/services/lti_h.py

```diff
@@ -16,7 +16,6 @@ class LTIHService:
     """
 
     def __init__(self, _context, request):
-        self._lti_user = request.lti_user
         self._h_user = request.lti_user.h_user
```

Comment:
This is used literally once lower down. Is there a reason for it being set here? Ditto the `_application_instance_service`.

One possibility is that the sequencing matters?

Reply:
I don't know that the rationale was was. Maybe not having the full `request` as an attribute?